test_unstructured_api_tools/api/test_file_apis.py

```diff
@@ -377,7 +371,7 @@ def test_process_file_2(
             None,
             marks=pytest.mark.xfail,
         ),
-        ([FILE_DOCX, FILE_IMAGE], None, RESPONSE_SCHEMA_ISD, 200, False, None, None),
+        ([FILE_DOCX, FILE_IMAGE], None, RESPONSE_SCHEMA_ISD, 406, False, None, None),
```

Comment:
Why do we now expect 406 on these? From the PR and issue descriptions I was assuming this would only make changes such that a single file can be explicitly requested for a multipart/form data response (and succeed).

Reply:
This is correct, because this particular [test pipeline_api #3 ](https://github.com/Unstructured-IO/unstructured-api-tools/blob/main/test_unstructured_api_tools/pipeline-test-project/pipeline-notebooks/pipeline-process-file-3.ipynb) has a default `response_type` of `text/csv`.

I.e., that is a reasonable pipeline_api definition that user may have created.

Reply:
Ah, okay, that makes perfect sense. i.e. this isn't a default that is happening _everywhere_. Okay great!